Composer/packages/adaptive-form/src/components/ErrorInfo.tsx

```diff
@@ -9,7 +9,7 @@ const ErrorInfo: React.FC<FallbackProps> = ({ componentStack, error }) => (
   <div style={{ marginRight: '20px' }}>
     <MessageBar isMultiline={false} messageBarType={MessageBarType.error}>
       <p style={{ whiteSpace: 'pre-wrap' }}>
-        <strong>{formatMessage({ default: 'An error occured in the form editor!', id: 'ErrorInfo_part1' })}</strong>
+        <strong>{formatMessage({ default: 'An error occurred in the form editor!', id: 'ErrorInfo_part1' })}</strong>
```

Comment:
You changed a formatMessage string, so make sure you are checking in an updated en-US.json.